MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -107,6 +71,41 @@ class RouteManeuverViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         turnArrowView.backgroundColor = .clear
+        streetLabel.availableBounds = { CGRect(origin: .zero, size: self.maximumAvailableStreetLabelSize) }
```

Comment:
Does `self` need to be captured strongly or weakly here? Or is this a nonescaping closure?